dotnet/SimpleSauce.Test/SauceSessionTests.cs

```diff
@@ -1,68 +1,47 @@
-using System;
-using System.Reflection;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using Newtonsoft.Json;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+using OpenQA.Selenium.Safari;
 using Simple.Sauce;
 
 namespace SimpleSauce.Test
 {
     [TestClass]
     public class SauceSessionTests : BaseTest
     {
-        private Mock<ISauceRemoteDriver> _dummyDriver;
+        private Mock<IDriverFactory> _driverFactory;
+        private Mock<SauceOptions> _mockOptions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field '_mockOptions'.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4748224)